cmd/logger/target/http/http.go

```diff
@@ -193,6 +199,16 @@ func WithAuthToken(authToken string) Option {
 // WithTransport adds a custom transport with custom timeouts and tuning.
 func WithTransport(transport *http.Transport) Option {
 	return func(t *Target) {
+		if t.clientCert != "" && t.clientKey != "" {
+			ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+			defer cancel()
+			c, err := certs.NewManager(ctx, t.clientCert, t.clientKey, tls.LoadX509KeyPair)
+			if err != nil {
+				logger.LogIf(ctx, fmt.Errorf("failed to load client key and cert, please check your endpoint configuration: %s",
+					err.Error()))
+			}
+			transport.TLSClientConfig.GetClientCertificate = c.GetClientCertificate
```

Comment:
We should re-do this a bit by making the WithTransport to be already populated with the client keys @nitisht such that the error is not masked here

Have a new function such that it takes client keys for mTLS like 

```
newCustomHTTPTransportWithClientCerts(clientCert, clientKey string)
```

This would return the transport which can be automatically used here, this can be used during the AddAuditTargets()